ops/services/app_service/metabase/_var.tf

```diff
@@ -45,6 +44,11 @@ variable "ai_instrumentation_key" {
   sensitive   = true
 }
 
+variable "app_settings_overrides" {
+  description = "override values for app_settings"
+  default     = {}
```

Comment:
Should be `sensitive = true`, no?

Reply:
_Technically_ no, because there could be a case where we use this config but don't pass in a sensitive value, but really yes, since I think that's the only way we're going to use this. It would inherit the `sensitive` flag from the password being passed in as part of it, but we can just make it explicit if you think it reads better.

Reply:
Oh, if the password data is going to stay successfully tagged without our intervention, then I'm fine with not tagging it—just don't want terraform 0.15 to suddenly break on us.